roles/openshift_node/tasks/upgrade/bootstrap_changes.yml

```diff
@@ -64,7 +64,7 @@
       - "{{ openshift.common.config_base }}/node/certificates"
     - key: kubeletArguments.feature-gates
       value:
-      - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+      - "RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true{% if (openshift_node_use_ephemeralstorage) %},LocalStorageCapacityIsolation={{openshift_node_use_ephemeralstorage}}{% endif %}"
```

Comment:
Generate this string from `l_openshift_feature_gates`

Reply:
+1